riscv/riscv.mk.in

```diff
@@ -38,6 +38,8 @@ riscv_hdrs = \
 	extension.h \
 	rocc.h \
 	insn_template.h \
+	insn_template_fast.h \
+	insn_template_slow.h \
```

Comment:
Is it "fast" and "slow", or "fast" and "logged"?

Reply:
Fixed to fast/logged. I think fast/slow is confusing in this context